prow/github/types.go

```diff
@@ -813,10 +813,18 @@ func (pe PushEvent) Branch() string {
 	return ref
 }
 
-// Commit represents general info about a commit.
-type Commit struct {
+// HeadCommit represents a git commit in a GitHub PushEvent.
+type HeadCommit struct {
```

Comment:
Can we please keep the name of the type?

Reply:
We have a new type name also called Commit. The current name is the same as go-github.

Reply:
Renamed. PTAL~ @alvaroaleman

Reply:
Now the `RepositoryCommit` same as https://github.com/google/go-github/blob/b47a3f75fc03ed477be4f04813180deb31407570/github/repos_commits.go#L18 and `GitCommit` same as https://github.com/google/go-github/blob/b47a3f75fc03ed477be4f04813180deb31407570/github/git_commits.go#L24.

Reply:
I don't know should I keep the old comment for `Commit`, but I think the current comment is more accurate.

Reply:
@alvaroaleman ping~